app/bundles/SmsBundle/EventListener/StopSubscriber.php

```diff
@@ -24,6 +24,14 @@ class StopSubscriber implements EventSubscriberInterface
      */
     private $doNotContactModel;
 
+    /**
+     * StopSubscriber constructor.
+     */
+    public function __construct($doNotContactModel)
```

Comment:
@mabumusa1 Code looks good!

Minor nit: would you mind changing this to `public function __construct(DoNotContactModel $doNotContactModel)` so that we benefit from PHP's [Type declarations](https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration)? Came across quite some bugs recently which could've been prevented if some functions had type declarations; would be great if we could add those from now on as Mautic is now PHP 7+ only 😊 thanks!!

Reply:
@dennisameling I added this type declaration, thanks for the tip